onnx2torch/node_converters/max_pool.py

```diff
@@ -54,6 +54,9 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:
         dilation=dilation,
         ceil_mode=ceil_mode == 1,
     )
+    if padding_module is not None:
+        padding_module.constant_value = float('-inf')
```

Comment:
You replace constant_value with -inf. Why?

Reply:
because max pool operation must ignore padded part

Reply:
Please add comments about it